plugins/generic/pln/PLNPlugin.inc.php

```diff
@@ -163,7 +163,7 @@ function getHandlerPath() {
 	 * @return string
 	 */
 	function getTemplatePath() {
-		return parent::getTemplatePath() . DIRECTORY_SEPARATOR . 'templates';
+		return parent::getTemplatePath() . DIRECTORY_SEPARATOR . 'templates/';
```

Comment:
I'm ambivalent about `DIRECTORY_SEPARATOR` vs. `/`, but suggest keeping consistent on any given line of code :)